src/views/app/service-management/create-service.tsx

```diff
@@ -0,0 +1,288 @@
+import { ParamListBase, useFocusEffect, useIsFocused, useNavigation } from '@react-navigation/native';
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { Icon } from '@rneui/themed';
+import { Formik, FormikConfig } from 'formik';
+import { Box, FormControl, HStack, VStack } from 'native-base';
+import React from 'react';
+import ButtonComponent from '../../../components/atoms/button';
+import { InputComponent } from '../../../components/atoms/input';
+import { SelectComponent, SelectItemComponent } from '../../../components/atoms/select';
+import { DateTimePickerComponent } from '../../../components/composite/date-picker';
+import ViewWrapper from '../../../components/layout/viewWrapper';
+import { THEME_CONFIG } from '../../../config/appConfig';
+import useModal from '../../../hooks/modal/useModal';
+import useRole from '../../../hooks/role';
+import { useCreateServiceMutation, useUpdateServiceMutation } from '../../../store/services/services';
+import { IAllService, ICreateServicePayload } from '../../../store/types';
+import Utils from '../../../utils';
+import { CreateServiceSchema } from '../../../utils/schemas';
+
+const tags: any = [
+    { id: 'cozasunday', value: 'COZASunday' },
+    { id: 'cozawednesday', value: 'COZAWednesday' },
+    { id: 'dpe', value: 'DPE' },
+    { id: 'home-training', value: 'Home Training' },
+    { id: 'leaders-meeting', value: 'Leaders Meeting' },
+    { id: '12dg', value: '12DG' },
+    { id: '7dg', value: '7DG' },
+];
+
+const CreateServiceManagement: React.FC<NativeStackScreenProps<ParamListBase>> = ({ navigation }) => {
+    const { navigate } = navigation;
+    const { setOptions } = useNavigation();
+
+    const { setModalState } = useModal();
+
+    const [createService, { isError, isLoading, isSuccess, error, data, reset }] = useCreateServiceMutation();
+
+    const onSubmit: FormikConfig<ICreateServicePayload>['onSubmit'] = async (values, { resetForm }) => {
+        const clockInStartTime = Utils.concatDateTimeToEpoc(values.startDate, values.clockinTime);
+        const coordinates = {
+            long: 7.505862981744857,
+            lat: 9.005452823370131,
```

Comment:
If this is to be hardcoded, then put it in an `enum` in `Utils` folder, under `types`.